sklearn/decomposition/tests/test_factor_analysis.py

```diff
@@ -83,3 +83,29 @@ def test_factor_analysis():
         precision = fa.get_precision()
         assert_array_almost_equal(np.dot(cov, precision),
                                   np.eye(X.shape[1]), 12)
+
+    # test rotation
+    n_comps = 2
+
+    results = []
+    for method in (None, "varimax", 'quartimax'):
+        fa_var = FactorAnalysis(n_components=n_comps, rotation=method)
+        results.append(fa_var.fit_transform(X))
+    for a, b in ((0, 1), (0, 2), (1, 2)):
```

Comment:
make the outer collection a list to improve readability `[(0, 1), (0, 2) ...]`

Reply:
but it would be much more readable to use `itertools.combinations([None, 'varimax', 'quartimax'], 2)` and have stored the results by name